plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -1,22 +1,37 @@
-use std::str::FromStr;
-
 use alumet::{
     metrics::{MetricCreationError, TypedMetricId},
     plugin::AlumetPluginStart,
     units::{PrefixedUnit, Unit},
 };
+use anyhow::{Context, Result};
+use std::str::FromStr;
 
 pub(crate) const CGROUP_MAX_TIME_COUNTER: u64 = u64::MAX;
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct CgroupV2Metric {
+    /// Name of a Kubernetes pod.
     pub name: String,
+    /// Unique identification of a Kubernetes pod.
     pub uid: String,
+    /// Resources isolation of a Kubernetes pod.
     pub namespace: String,
+    /// Kubernetes pod node.
     pub node: String,
+    /// Total CPU usage time by the cgroup.
     pub time_used_tot: u64,
+    /// CPU in user mode usage time by the cgroup.
     pub time_used_user_mode: u64,
+    /// CPU in system mode usage time by the cgroup.
     pub time_used_system_mode: u64,
+    /// Anonymous used memory, corresponding to running process and various allocated memory.
+    pub anon_used_mem: u64,
```

Comment:
Rename to: `memory_anonymous`. Same for other memory values: put "memory" first (so it's easier to distinguish between cpu time and memory fields).